cores/rp2040/RP2040.h

```diff
@@ -118,89 +142,108 @@ class _MFIFO {
 
 class RP2040;
 extern RP2040 rp2040;
-class RP2040 {
+class RP2040
+{
 public:
-    RP2040() {
+    RP2040()
+    {
         _epoch = 0;
         // Enable SYSTICK exception
         exception_set_exclusive_handler(SYSTICK_EXCEPTION, _SystickHandler);
         systick_hw->csr = 0x7;
         systick_hw->rvr = 0x00FFFFFF;
     }
 
-    ~RP2040() { /* noop */ }
-
+    ~RP2040()
+    { /* noop */
+    }
 
     // Convert from microseconds to PIO clock cycles
-    static int usToPIOCycles(int us) {
+    static int usToPIOCycles(float us)
+    {
         // Parenthesis needed to guarantee order of operations to avoid 32bit overflow
-        return (us * (clock_get_hz(clk_sys) / 1000000));
+        float x = (us * (clock_get_hz(clk_sys) / 1000000));
```

Comment:
Do we need to use a floating million to ensure float and not integer division here?  `(float)1000000.0`  Because `clock_get_hz` returns `int` and `10000000` is int, and they're evaluated first due to parenthesis, I think so.  OTW `/` is integer division and not float.

Reply:
Didn't really think about that and I'm not too sure how the division will be done but it wouldn't hurt to add it just in case